### PR TITLE
Add FX Graph Mode Post Training Dynamic Quantization Tutorial

### PR DESCRIPTION
**Pull request by jerryzh168**

### CODE REVIEW

File: prototype_source/README.txt [hunk 1]
```diff
@@ -26,4 +26,8 @@ Prototype Tutorials
 
 7. fx_graph_mode_static_quantization.py
 	   FX Graph Mode Post Training Static Quantization
-	   https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static_tutorial.html
\ No newline at end of file
+	   https://pytorch.org/tutorials/prototype/fx_graph_mode_ptq_static_tutorial.html
+
+7. fx_graph_mode_dynamic_quantization.py
```
Comment (jlin27) [inline prototype_source/README.txt:31]: Update to 8.

File: prototype_source/fx_graph_mode_ptq_dynamic.py [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+
+In this tutorial, we’ll apply dynamic quantization to an LSTM-based next word-prediction model, 
+closely following the word language model from the PyTorch examples. 
+We will copy the code from `Dynamic Quantization on an LSTM Word Language Model <https://pytorch.org/tutorials/advanced/dynamic_quantization_tutorial.html>`_ 
+and omit the descriptions.
+
+"""
+
+
+###################################################
```
Comment (jlin27) [inline prototype_source/fx_graph_mode_ptq_dynamic.py:35]: For some reason the header is not rendering properly when I built it locally:
image

File: prototype_source/README.txt (hunk 1 above)
Comment (jlin27) [inline prototype_source/README.txt:31]: Update the numbering to 8.

**Review by jlin27 — CHANGES_REQUESTED** [review]
Thanks for putting this together Jerry. I left 2 items to check on. Also just adding reminder in this comment to update todos before release.

File: prototype_source/fx_graph_mode_ptq_dynamic.py (hunk 2 above)
Comment (jerryzh168) [reply prototype_source/fx_graph_mode_ptq_dynamic.py:35]: oh there is one missing '-'

**Comment by jerryzh168** [comment]
thanks, created an issue to track it https://github.com/pytorch/tutorials/issues/1289

**Review by jlin27 — APPROVED** [review]

**Comment by jerryzh168** [comment]
@jlin27 @brianjo could you merge this?

**Pull request merged**